shell/browser/api/electron_api_cookies.cc

```diff
@@ -100,25 +101,12 @@ namespace electron::api {
 
 namespace {
 
-// Returns whether |domain| matches |filter|.
-bool MatchesDomain(std::string filter, const std::string& domain) {
-  // Add a leading '.' character to the filter domain if it doesn't exist.
-  if (net::cookie_util::DomainIsHostOnly(filter))
-    filter.insert(0, ".");
-
-  std::string sub_domain(domain);
+bool DomainIs(std::string_view host, const std::string_view domain) {
```

Comment:
(note: the changes of argument order and naming is a little confusing at first glance. What's going on here is they're now consistent with the order and naming of `url::DomainIs()`.)